db/migrate/20200423070105_create_lots.rb

```diff
@@ -0,0 +1,16 @@
+class CreateLots < ActiveRecord::Migration[6.0]
+  def change
+    create_table :lots do |t|
+      t.string :title
```

Comment:
Add validation to the database `null: false` on fields which is required
Here and in all migrations